### DISCUSSION TITLE
The user named '~yannick' does not have any PPA

### DISCUSSION THREAD

**Discussion by AmineBarrak** [abraunegg/onedrive#2504] [General]
I am having this error when i try to install onedrive on ubuntu 20.04:
sudo add-apt-repository --remove ppa:yannick/onedrive
Cannot add PPA: 'ppa:~yannick/ubuntu/onedrive'.
The user named '~yannick' does not have any PPA

**Comment by abraunegg** [comment]
@AmineBarrak 
That PPA was depreciated quite a few years ago. 

You can safely ignore the response, as you did not have it added to your system, thus that error response is 100% normal.
